src/main/java/camp/nextstep/edu/level1/lotto/lotto/Lotto.java

```diff
@@ -5,12 +5,23 @@
 
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 
 public class Lotto {
+    private static final int LOTTO_START_NUMBER = 1;
+    private static final int LOTTO_END_NUMBER = 45;
+    private static final int LOTTO_RANGE = 6;
     private static final long LOTTO_PRICE = 1000;
     private static final int WINNING_NUMBER_COUNT = 6;
     private static final String WINNING_NUMBER_DELIMITER = ",";
+    private static final List<Integer> LOTTO_NUMBER_PRESET = new ArrayList<>();
```

Comment:
```suggestion
    private static final List<LottoNumber> LOTTO_NUMBER_PRESET = new ArrayList<>();
```
해당 부분도 LottoNumber로 관리하는 건 어떤가요?

Reply:
오... 넵 적용하겠습니다.

단순히 Lotto 객체에서 생성되니 잘 된다고만 생각했었던 것 같네요.
감사합니다